### ISSUE TITLE
Alpha_Zero 수정사항

### ISSUE THREAD

**Issue by kekmodel** [reinforcement-learning-kr/alpha_omok#50]
### Debug to backup value incorrectly
- value가 음양이 교대로 백업되지 않았던 치명적인 버그 수정함
- 이제 학습이 되는 것으로 보여지며 현재 테스트 중

### Add module tensorboardX to use tensorboard
- 학습 관찰을 위해 tensorboardX 모듈 추가
- ***pip install tensorboardX*** 설치 필요
- tensorboardX를 사용한 코드 실행시 코드가 포함된 폴더에 runs라는 폴더가 자동 생성됨
- 그 위치에서 **tensorboard --logdir runs**(=폴더경로) 으로 실행하면 웹 브라우저로 실시간 관찰 가능

#### Add symmetry_choice function
- 8대칭 변형을 해주는 함수 utils.py 에 추가

#### Add code to control tau so pi is not nan
- tau가 지수여서 N이 너무 큰경우 nan이 발생할 수 있음
- 실험을 통해 nan이 발생하는 크기에서 tau를 조절하도록 코드 추가

#### Change memory storage method
- 데이터셋 저장시 흑위주, 백위주로 편향되어 저장되는 것 수정
- shuffle을 하여도 영향을 받을 수 있을 것 같았음
- 타임스텝 순서대로 저장되도록 수정하였음

#### Change neural network initialization method
- None끼리 붙여봤을때 승률 젤 좋은 놈으로 세팅하였음
- kaiming_normal
- ***pytorch 0.4 설치 필요***

#### Cleanup codebase
- 지저분한 코드들 함수 만들어서 정리
- ***pytorch 0.4 설치 필요***
